app/models/applicant.rb

```diff
@@ -47,4 +47,21 @@ def age
   def child?
     age < 16
   end
+
+  def receives_financial_support?
+    return false if bank_transactions.empty?
+
+    bank_transactions.map(&:transaction_type_id).include?(TransactionType.find_by(name: 'friends_or_family')&.id)
```

Comment:
we have a public class method on the bank_transaction model  `self.for_type(transaction_type_name)` which should simplify this to `BankTransactions.for_type('friends_or_family').present?`